### ISSUE TITLE
Gstreamer: gst-plugins-espeak (gst-inspect-1.0:14426): GLib-GObject-CRITICAL **: g_value_set_uint: assertion 'G_VALUE_HOLDS_UINT (value)' failed

### ISSUE THREAD

**Issue by bossjones** [bossjones/scarlett-dbus-poc#6]
When running `gst-inspect-1.0 espeak`, kept coming across the following error message: `(gst-inspect-1.0:14426): GLib-GObject-CRITICAL **: g_value_set_uint: assertion 'G_VALUE_HOLDS_UINT (value)' failed`

The fact that this is a critical error, I figured that is might be causing more harm than good for the latest iteration of the ScarlettSpeaker class.

I went on IRC, specifically freenode to the `#gstreamer` channel to ask whoever was in there if they knew what the error message meant. Luckily, `__tim` wasn't too busy, and decided to take a quick look at the old and unmaintained `gst-plugins-espeak` source code. Couple screen shots from our convo along with the solution coming up below:

![screen shot 2016-04-30 at 12 15 29 pm](https://cloud.githubusercontent.com/assets/709872/14937316/c1c3829e-0ed1-11e6-8725-6451d42e76d9.png)
![screen shot 2016-04-30 at 12 21 37 pm](https://cloud.githubusercontent.com/assets/709872/14937318/c81c7b0a-0ed1-11e6-9235-73ee8e5cf6fd.png)
![screen shot 2016-04-30 at 12 37 34 pm](https://cloud.githubusercontent.com/assets/709872/14937319/cd74636a-0ed1-11e6-9936-71e58beee9f8.png)

gst-inspect-1.0 espeak output:

```
± |feature-new-gst-and-pocketsphinx U:11 ?:50 ✗| → gst-inspect-1.0 espeak
Factory Details:
  Rank                     none (0)
  Long-name                eSpeak as a sound source
  Klass                    Source/Audio
  Description              Use eSpeak library as a sound source for GStreamer
  Author                   Aleksey Lim <alsroot@sugarlabs.org>

Plugin Details:
  Name                     espeak
  Description              Uses eSpeak library as a sound source for GStreamer
  Filename                 /home/pi/.virtualenvs/scarlett-dbus-poc/lib/gstreamer-1.0/libgstespeak.so
  Version                  0.4.0
  License                  LGPL
  Source module            gst-plugins-espeak
  Binary package           gst-plugins-espeak
  Origin URL               http://sugarlabs.org/go/DevelopmentTeam/gst-plugins-espeak

GObject
 +----GInitiallyUnowned
       +----GstObject
             +----GstElement
                   +----GstBaseSrc
                         +----GstEspeak

Implemented Interfaces:
  GstURIHandler

Pad Templates:
  SRC template: 'src'
    Availability: Always
    Capabilities:
      ANY


Element Flags:
  no flags set

Element Implementation:
  Has change_state() function: gst_base_src_change_state

Element has no clocking capabilities.

URI handling capabilities:
  Element can act as source.
  Supported URI protocols:
    espeak

Pads:
  SRC: 'src'
    Pad Template: 'src'

Element Properties:
  name                : The name of the object
                        flags: readable, writable
                        String. Default: "espeak0"
  parent              : The parent of the object
                        flags: readable, writable
                        Object of type "GstObject"
  blocksize           : Size in bytes to read per buffer (-1 = default)
                        flags: readable, writable
                        Unsigned Integer. Range: 0 - 4294967295 Default: 2205
  num-buffers         : Number of buffers to output before sending EOS (-1 = unlimited)
                        flags: readable, writable
                        Integer. Range: -1 - 2147483647 Default: -1
  typefind            : Run typefind before negotiating
                        flags: readable, writable
                        Boolean. Default: false
  do-timestamp        : Apply current stream time to buffers
                        flags: readable, writable
                        Boolean. Default: false
  text                : Text to pronounce
                        flags: readable, writable
                        String. Default: null
  pitch               : Pitch adjustment

(gst-inspect-1.0:14426): GLib-GObject-CRITICAL **: g_value_set_uint: assertion 'G_VALUE_HOLDS_UINT (value)' failed
                        flags: readable, writable
                        Integer. Range: -100 - 100 Default: 0
  rate                : Speed in words per minute

(gst-inspect-1.0:14426): GLib-GObject-CRITICAL **: g_value_set_uint: assertion 'G_VALUE_HOLDS_UINT (value)' failed
                        flags: readable, writable
                        Integer. Range: -100 - 100 Default: 0
  voice               : Current voice
                        flags: readable, writable
                        String. Default: "default"
  gap                 : Word gap
                        flags: readable, writable
                        Unsigned Integer. Range: 0 - 2147483647 Default: 0
  track               : Track espeak events
                        flags: readable, writable
                        Unsigned Integer. Range: 0 - 2147483647 Default: 0
  voices              : List of voices
                        flags: readable
                        Boxed pointer of type "GValueArray"
  caps                : Caps describing the format of the data
                        flags: readable
                           audio/x-raw
                                      format: S16LE
                                      layout: interleaved
                                        rate: 22050
                                    channels: 1
```

Output from recompiling gst-plugins-espeak:

```
checking for a BSD-compatible install... /usr/bin/install -c
checking whether build environment is sane... yes
checking for a thread-safe mkdir -p... /bin/mkdir -p
checking for gawk... no
checking for mawk... mawk
checking whether make sets $(MAKE)... yes
checking whether to enable maintainer-specific portions of Makefiles... no
checking whether make supports nested variables... yes
checking for gcc... gcc
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables...
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether gcc accepts -g... yes
checking for gcc option to accept ISO C89... none needed
checking for style of include used by make... GNU
checking dependency style of gcc... gcc3
checking build system type... x86_64-unknown-linux-gnu
checking host system type... x86_64-unknown-linux-gnu
checking how to print strings... printf
checking for a sed that does not truncate output... /bin/sed
checking for grep that handles long lines and -e... /bin/grep
checking for egrep... /bin/grep -E
checking for fgrep... /bin/grep -F
checking for ld used by gcc... /usr/bin/ld
checking if the linker (/usr/bin/ld) is GNU ld... yes
checking for BSD- or MS-compatible name lister (nm)... /usr/bin/nm -B
checking the name lister (/usr/bin/nm -B) interface... BSD nm
checking whether ln -s works... yes
checking the maximum length of command line arguments... 1572864
checking whether the shell understands some XSI constructs... yes
checking whether the shell understands "+="... yes
checking how to convert x86_64-unknown-linux-gnu file names to x86_64-unknown-linux-gnu format... func_convert_file_noop
checking how to convert x86_64-unknown-linux-gnu file names to toolchain format... func_convert_file_noop
checking for /usr/bin/ld option to reload object files... -r
checking for objdump... objdump
checking how to recognize dependent libraries... pass_all
checking for dlltool... no
checking how to associate runtime and link libraries... printf %s\n
checking for ar... ar
checking for archiver @FILE support... @
checking for strip... strip
checking for ranlib... ranlib
checking command to parse /usr/bin/nm -B output from gcc object... ok
checking for sysroot... no
checking for mt... mt
checking if mt is a manifest tool... no
checking how to run the C preprocessor... gcc -E
checking for ANSI C header files... yes
checking for sys/types.h... yes
checking for sys/stat.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for memory.h... yes
checking for strings.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for unistd.h... yes
checking for dlfcn.h... yes
checking for objdir... .libs
checking if gcc supports -fno-rtti -fno-exceptions... no
checking for gcc option to produce PIC... -fPIC -DPIC
checking if gcc PIC flag -fPIC -DPIC works... yes
checking if gcc static flag -static works... yes
checking if gcc supports -c -o file.o... yes
checking if gcc supports -c -o file.o... (cached) yes
checking whether the gcc linker (/usr/bin/ld -m elf_x86_64) supports shared libraries... yes
checking whether -lc should be explicitly linked in... no
checking dynamic linker characteristics... GNU/Linux ld.so
checking how to hardcode library paths into programs... immediate
checking whether stripping libraries is possible... yes
checking if libtool supports shared libraries... yes
checking whether to build shared libraries... yes
checking whether to build static libraries... yes
checking for pkg-config... /usr/bin/pkg-config
checking pkg-config is at least version 0.9.0... yes
checking for GST... yes
checking for GST_AUDIO... yes
checking for espeak_Initialize in -lespeak... yes
configure: creating ./config.status
config.status: creating Makefile
config.status: creating m4/Makefile
config.status: creating src/Makefile
config.status: creating config.h
config.status: config.h is unchanged
config.status: executing depfiles commands
config.status: executing libtool commands
make  all-recursive
make[1]: Entering directory '/home/pi/dev/bossjones-github/scarlett-dbus-poc/gst-plugins-espeak-0.4.0'
Making all in m4
make[2]: Entering directory '/home/pi/dev/bossjones-github/scarlett-dbus-poc/gst-plugins-espeak-0.4.0/m4'
make[2]: Nothing to be done for 'all'.
make[2]: Leaving directory '/home/pi/dev/bossjones-github/scarlett-dbus-poc/gst-plugins-espeak-0.4.0/m4'
Making all in src
make[2]: Entering directory '/home/pi/dev/bossjones-github/scarlett-dbus-poc/gst-plugins-espeak-0.4.0/src'
  CC     libgstespeak_la-espeak.lo
In file included from /usr/include/gstreamer-1.0/gst/gst.h:54:0,
                 from espeak.c:21:
espeak.c: In function ‘espeak_out’:
espeak.c:287:16: warning: format ‘%d’ expects argument of type ‘int’, but argument 9 has type ‘gsize {aka long unsigned int}’ [-Wformat=]
     GST_DEBUG ("[%p] size_to_play=%d", self, size_to_play);
                ^
/usr/include/gstreamer-1.0/gst/gstinfo.h:541:31: note: in definition of macro ‘GST_CAT_LEVEL_LOG’
         (GObject *) (object), __VA_ARGS__);    \
                               ^
espeak.c:287:5: note: in expansion of macro ‘GST_DEBUG’
     GST_DEBUG ("[%p] size_to_play=%d", self, size_to_play);
     ^
espeak.c: In function ‘espeak_get_voices’:
espeak.c:445:5: warning: ‘g_value_array_copy’ is deprecated: Use 'GArray' instead [-Wdeprecated-declarations]
     return g_value_array_copy (espeak_voices);
     ^
In file included from /usr/include/glib-2.0/glib-object.h:35:0,
                 from /usr/include/gstreamer-1.0/gst/gstenumtypes.h:7,
                 from /usr/include/gstreamer-1.0/gst/gst.h:31,
                 from espeak.c:21:
/usr/include/glib-2.0/gobject/gvaluearray.h:73:14: note: declared here
 GValueArray* g_value_array_copy      (const GValueArray *value_array);
              ^
In file included from /usr/include/gstreamer-1.0/gst/gst.h:54:0,
                 from espeak.c:21:
espeak.c: In function ‘process’:
espeak.c:500:24: warning: format ‘%d’ expects argument of type ‘int’, but argument 9 has type ‘gsize {aka long unsigned int}’ [-Wformat=]
             GST_DEBUG ("[%p] context->text_offset=%d context->text_len=%d",
                        ^
/usr/include/gstreamer-1.0/gst/gstinfo.h:541:31: note: in definition of macro ‘GST_CAT_LEVEL_LOG’
         (GObject *) (object), __VA_ARGS__);    \
                               ^
espeak.c:500:13: note: in expansion of macro ‘GST_DEBUG’
             GST_DEBUG ("[%p] context->text_offset=%d context->text_len=%d",
             ^
espeak.c:500:24: warning: format ‘%d’ expects argument of type ‘int’, but argument 10 has type ‘gsize {aka long unsigned int}’ [-Wformat=]
             GST_DEBUG ("[%p] context->text_offset=%d context->text_len=%d",
                        ^
/usr/include/gstreamer-1.0/gst/gstinfo.h:541:31: note: in definition of macro ‘GST_CAT_LEVEL_LOG’
         (GObject *) (object), __VA_ARGS__);    \
                               ^
espeak.c:500:13: note: in expansion of macro ‘GST_DEBUG’
             GST_DEBUG ("[%p] context->text_offset=%d context->text_len=%d",
             ^
espeak.c: In function ‘init’:
espeak.c:566:9: warning: ‘g_mutex_new’ is deprecated [-Wdeprecated-declarations]
         process_lock = g_mutex_new ();
         ^
In file included from /usr/include/glib-2.0/glib.h:107:0,
                 from espeak.c:20:
/usr/include/glib-2.0/glib/deprecated/gthread.h:277:17: note: declared here
 GMutex *        g_mutex_new             (void);
                 ^
espeak.c:567:9: warning: ‘g_cond_new’ is deprecated [-Wdeprecated-declarations]
         process_cond = g_cond_new ();
         ^
In file included from /usr/include/glib-2.0/glib.h:107:0,
                 from espeak.c:20:
/usr/include/glib-2.0/glib/deprecated/gthread.h:281:17: note: declared here
 GCond *         g_cond_new              (void);
                 ^
espeak.c:568:9: warning: ‘g_thread_create’ is deprecated: Use 'g_thread_new' instead [-Wdeprecated-declarations]
         process_tid = g_thread_create (process, NULL, FALSE, NULL);
         ^
In file included from /usr/include/glib-2.0/glib.h:107:0,
                 from espeak.c:20:
/usr/include/glib-2.0/glib/deprecated/gthread.h:104:10: note: declared here
 GThread *g_thread_create       (GThreadFunc       func,
          ^
espeak.c:583:9: warning: ‘g_value_array_new’ is deprecated: Use 'GArray' instead [-Wdeprecated-declarations]
         espeak_voices = g_value_array_new (count);
         ^
In file included from /usr/include/glib-2.0/glib-object.h:35:0,
                 from /usr/include/gstreamer-1.0/gst/gstenumtypes.h:7,
                 from /usr/include/gstreamer-1.0/gst/gst.h:31,
                 from espeak.c:21:
/usr/include/glib-2.0/gobject/gvaluearray.h:67:14: note: declared here
 GValueArray* g_value_array_new      (guint   n_prealloced);
              ^
espeak.c:586:13: warning: ‘g_value_array_new’ is deprecated: Use 'GArray' instead [-Wdeprecated-declarations]
             GValueArray *voice = g_value_array_new (2);
             ^
In file included from /usr/include/glib-2.0/glib-object.h:35:0,
                 from /usr/include/gstreamer-1.0/gst/gstenumtypes.h:7,
                 from /usr/include/gstreamer-1.0/gst/gst.h:31,
                 from espeak.c:21:
/usr/include/glib-2.0/gobject/gvaluearray.h:67:14: note: declared here
 GValueArray* g_value_array_new      (guint   n_prealloced);
              ^
espeak.c:591:13: warning: ‘g_value_array_append’ is deprecated: Use 'GArray' instead [-Wdeprecated-declarations]
             g_value_array_append (voice, &name);
             ^
In file included from /usr/include/glib-2.0/glib-object.h:35:0,
                 from /usr/include/gstreamer-1.0/gst/gstenumtypes.h:7,
                 from /usr/include/gstreamer-1.0/gst/gst.h:31,
                 from espeak.c:21:
/usr/include/glib-2.0/gobject/gvaluearray.h:80:14: note: declared here
 GValueArray* g_value_array_append      (GValueArray *value_array,
              ^
espeak.c:600:13: warning: ‘g_value_array_append’ is deprecated: Use 'GArray' instead [-Wdeprecated-declarations]
             g_value_array_append (voice, &lang);
             ^
In file included from /usr/include/glib-2.0/glib-object.h:35:0,
                 from /usr/include/gstreamer-1.0/gst/gstenumtypes.h:7,
                 from /usr/include/gstreamer-1.0/gst/gst.h:31,
                 from espeak.c:21:
/usr/include/glib-2.0/gobject/gvaluearray.h:80:14: note: declared here
 GValueArray* g_value_array_append      (GValueArray *value_array,
              ^
espeak.c:606:13: warning: ‘g_value_array_append’ is deprecated: Use 'GArray' instead [-Wdeprecated-declarations]
             g_value_array_append (voice, &dialect);
             ^
In file included from /usr/include/glib-2.0/glib-object.h:35:0,
                 from /usr/include/gstreamer-1.0/gst/gstenumtypes.h:7,
                 from /usr/include/gstreamer-1.0/gst/gst.h:31,
                 from espeak.c:21:
/usr/include/glib-2.0/gobject/gvaluearray.h:80:14: note: declared here
 GValueArray* g_value_array_append      (GValueArray *value_array,
              ^
espeak.c:609:13: warning: ‘g_value_array_get_type’ is deprecated: Use 'GArray' instead [-Wdeprecated-declarations]
             g_value_init (&voice_value, G_TYPE_VALUE_ARRAY);
             ^
In file included from /usr/include/glib-2.0/glib-object.h:35:0,
                 from /usr/include/gstreamer-1.0/gst/gstenumtypes.h:7,
                 from /usr/include/gstreamer-1.0/gst/gst.h:31,
                 from espeak.c:21:
/usr/include/glib-2.0/gobject/gvaluearray.h:60:17: note: declared here
 GType           g_value_array_get_type       (void) G_GNUC_CONST;
                 ^
espeak.c:611:13: warning: ‘g_value_array_append’ is deprecated: Use 'GArray' instead [-Wdeprecated-declarations]
             g_value_array_append (espeak_voices, &voice_value);
             ^
In file included from /usr/include/glib-2.0/glib-object.h:35:0,
                 from /usr/include/gstreamer-1.0/gst/gstenumtypes.h:7,
                 from /usr/include/gstreamer-1.0/gst/gst.h:31,
                 from espeak.c:21:
/usr/include/glib-2.0/gobject/gvaluearray.h:80:14: note: declared here
 GValueArray* g_value_array_append      (GValueArray *value_array,
              ^
  CC     libgstespeak_la-gstespeak.lo
gstespeak.c: In function ‘gst_espeak_class_init’:
gstespeak.c:121:21: warning: ‘g_value_array_get_type’ is deprecated: Use 'GArray' instead [-Wdeprecated-declarations]
                     "List of voices", G_TYPE_VALUE_ARRAY,
                     ^
In file included from /usr/include/glib-2.0/glib-object.h:35:0,
                 from /usr/include/gstreamer-1.0/gst/gstenumtypes.h:7,
                 from /usr/include/gstreamer-1.0/gst/gst.h:31,
                 from gstespeak.c:35:
/usr/include/glib-2.0/gobject/gvaluearray.h:60:17: note: declared here
 GType           g_value_array_get_type       (void) G_GNUC_CONST;
                 ^
gstespeak.c: In function ‘gst_espeak_finalize’:
gstespeak.c:175:5: warning: ‘g_value_array_free’ is deprecated: Use 'GArray' instead [-Wdeprecated-declarations]
     g_value_array_free (self->voices);
     ^
In file included from /usr/include/glib-2.0/glib-object.h:35:0,
                 from /usr/include/gstreamer-1.0/gst/gstenumtypes.h:7,
                 from /usr/include/gstreamer-1.0/gst/gst.h:31,
                 from gstespeak.c:35:
/usr/include/glib-2.0/gobject/gvaluearray.h:70:7: note: declared here
 void  g_value_array_free      (GValueArray *value_array);
       ^
  CCLD   libgstespeak.la
make[2]: Leaving directory '/home/pi/dev/bossjones-github/scarlett-dbus-poc/gst-plugins-espeak-0.4.0/src'
make[2]: Entering directory '/home/pi/dev/bossjones-github/scarlett-dbus-poc/gst-plugins-espeak-0.4.0'
make[2]: Leaving directory '/home/pi/dev/bossjones-github/scarlett-dbus-poc/gst-plugins-espeak-0.4.0'
make[1]: Leaving directory '/home/pi/dev/bossjones-github/scarlett-dbus-poc/gst-plugins-espeak-0.4.0'
Making install in m4
make[1]: Entering directory '/home/pi/dev/bossjones-github/scarlett-dbus-poc/gst-plugins-espeak-0.4.0/m4'
make[2]: Entering directory '/home/pi/dev/bossjones-github/scarlett-dbus-poc/gst-plugins-espeak-0.4.0/m4'
make[2]: Nothing to be done for 'install-exec-am'.
make[2]: Nothing to be done for 'install-data-am'.
make[2]: Leaving directory '/home/pi/dev/bossjones-github/scarlett-dbus-poc/gst-plugins-espeak-0.4.0/m4'
make[1]: Leaving directory '/home/pi/dev/bossjones-github/scarlett-dbus-poc/gst-plugins-espeak-0.4.0/m4'
Making install in src
make[1]: Entering directory '/home/pi/dev/bossjones-github/scarlett-dbus-poc/gst-plugins-espeak-0.4.0/src'
make[2]: Entering directory '/home/pi/dev/bossjones-github/scarlett-dbus-poc/gst-plugins-espeak-0.4.0/src'
make[2]: Nothing to be done for 'install-exec-am'.
 /bin/mkdir -p '/home/pi/.virtualenvs/scarlett-dbus-poc/lib/gstreamer-1.0'
 /bin/bash ../libtool   --mode=install /usr/bin/install -c   libgstespeak.la '/home/pi/.virtualenvs/scarlett-dbus-poc/lib/gstreamer-1.0'
libtool: install: /usr/bin/install -c .libs/libgstespeak.so /home/pi/.virtualenvs/scarlett-dbus-poc/lib/gstreamer-1.0/libgstespeak.so
libtool: install: /usr/bin/install -c .libs/libgstespeak.lai /home/pi/.virtualenvs/scarlett-dbus-poc/lib/gstreamer-1.0/libgstespeak.la
libtool: finish: PATH="/home/pi/.virtualenvs/scarlett-dbus-poc/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/sbin" ldconfig -n /home/pi/.virtualenvs/scarlett-dbus-poc/lib/gstreamer-1.0
----------------------------------------------------------------------
Libraries have been installed in:
   /home/pi/.virtualenvs/scarlett-dbus-poc/lib/gstreamer-1.0

If you ever happen to want to link against installed libraries
in a given directory, LIBDIR, you must either use libtool, and
specify the full pathname of the library, or use the `-LLIBDIR'
flag during linking and do at least one of the following:
   - add LIBDIR to the `LD_LIBRARY_PATH' environment variable
     during execution
   - add LIBDIR to the `LD_RUN_PATH' environment variable
     during linking
   - use the `-Wl,-rpath -Wl,LIBDIR' linker flag
   - have your system administrator add LIBDIR to `/etc/ld.so.conf'

See any operating system documentation about shared libraries for
more information, such as the ld(1) and ld.so(8) manual pages.
----------------------------------------------------------------------
make[2]: Leaving directory '/home/pi/dev/bossjones-github/scarlett-dbus-poc/gst-plugins-espeak-0.4.0/src'
make[1]: Leaving directory '/home/pi/dev/bossjones-github/scarlett-dbus-poc/gst-plugins-espeak-0.4.0/src'
make[1]: Entering directory '/home/pi/dev/bossjones-github/scarlett-dbus-poc/gst-plugins-espeak-0.4.0'
make[2]: Entering directory '/home/pi/dev/bossjones-github/scarlett-dbus-poc/gst-plugins-espeak-0.4.0'
make[2]: Nothing to be done for 'install-exec-am'.
make[2]: Nothing to be done for 'install-data-am'.
make[2]: Leaving directory '/home/pi/dev/bossjones-github/scarlett-dbus-poc/gst-plugins-espeak-0.4.0'
make[1]: Leaving directory '/home/pi/dev/bossjones-github/scarlett-dbus-poc/gst-plugins-espeak-0.4.0'
```

Shell script used to re-compile `gst-plugins-espeak` ( Note I'm installing it directly into a python virtual environment ):

```
#!/usr/bin/env bash
# GOOD

set -o nounset
set -o errexit

export GSTREAMER=1.0
export PI_HOME=/home/pi
export MAIN_DIR=$PI_HOME/dev/bossjones-github/scarlett-dbus-poc
export VIRT_ROOT=$PI_HOME/.virtualenvs/scarlett-dbus-poc
export PKG_CONFIG_PATH=$VIRT_ROOT/lib/pkgconfig
export SCARLETT_CONFIG=$MAIN_DIR/tests/fixtures/.scarlett
export SCARLETT_HMM=/home/pi/.virtualenvs/scarlett-dbus-poc/share/pocketsphinx/model/en-us/en-us
export SCARLETT_LM=$MAIN_DIR/tests/fixtures/lm/1473.dic
export SCARLETT_DICT=$MAIN_DIR/tests/fixtures/dict/1473.dic

# for GST PLUGINS
export LD_LIBRARY_PATH=$VIRT_ROOT/lib
export GST_PLUGIN_PATH=$VIRT_ROOT/lib/gstreamer-$GSTREAMER

cd $MAIN_DIR && \
cd gst-plugins-espeak-0.4.0 && \
./configure --prefix=$VIRT_ROOT && \
make && \
make install && \
cd $MAIN_DIR
```

**Comment by bossjones** [comment]
TL;DR of fix - https://github.com/bossjones/bossjones-gst-plugins-espeak-0-4-0/pull/1

**Comment by bossjones** [comment]
fixed in https://github.com/bossjones/bossjones-gst-plugins-espeak-0-4-0/commit/9a64a037a975e925070bb3fddd0cb8eb56b384e9
